debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/olr/OpenLogReplicatorValueConverter.java

```diff
@@ -113,7 +114,7 @@ protected Object convertTimestampWithZone(Column column, Field fieldDefn, Object
 
     @Override
     protected Object convertTimestampWithLocalZone(Column column, Field fieldDefn, Object value) {
-        if (value instanceof Long) {
+        if (value instanceof Number) {
```

Comment:
This does not look safe. You check for `Number` but then you typecaset to `Long`

Reply:
Thanks, I'll take a look next week when back.